bee-ledger/src/workers/error.rs

```diff
@@ -49,9 +49,12 @@ pub enum Error {
     /// Non contiguous milestones.
     #[error("non contiguous milestones: tried to confirm {0} on top of {1}")]
     NonContiguousMilestones(u32, u32),
-    /// Merkle proof mismatch.
-    #[error("merkle proof mismatch on milestone {0}: computed {1} != provided {2}")]
-    MerkleProofMismatch(MilestoneIndex, String, String),
+    /// Confirmed merkle proof mismatch.
```

Comment:
I like @grtlr 's approach of putting `#[allow(missing_docs)]` on error enums, because the variants should be self-describing enough.